### PR TITLE
[ADD] time: init

### PR DESCRIPTION
**Pull request by oerp-odoo**
[BRANCH] feature/time-ala

### CODE REVIEW

**Review by Drsmartinkus — APPROVED** [review]
What about seconds? Or its out of scope?

File: footil/time.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+"""Helpers for time calculations."""
+
+HOUR_IN_MIN = 60
+DAY_IN_H = 24
```
Comment (SButko) [inline footil/time.py]:
```suggestion
DAY_IN_HOURS = 24
```
```suggestion
DAY_HOURS = 24
```

File: footil/time.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+"""Helpers for time calculations."""
+
+HOUR_IN_MIN = 60
```
Comment (SButko) [inline footil/time.py]: Abbreviation can be interpreted as 'how many hours in minutes' when plural not used :)
```suggestion
HOUR_IN_MINS = 60
```
```suggestion
HOUR_MINS = 60
```

**Review by SButko — APPROVED** [review]

**Comment by oerp-odoo** [comment]
> What about seconds? Or its out of scope?

Well, it was not needed for now. Plan is to make more generic converter, where you could for example convert from seconds to days or vise versa. But its left as todo for now.

**Pull request merged**